tests/testthat/test-aes-grouping.r

```diff
@@ -23,6 +23,12 @@ test_that("one group per combination of discrete vars", {
   expect_equal(groups(plot), 4)
 })
 
+test_that("no error for aes(groupS)", {
+  plot <- ggplot(df, aes(a, b, groupS=1)) + geom_point()
+  g <- ggplotGrob(plot)
+  expect_is(g, "grob")
```

Comment:
Could you test `add_group()` directly with `expect_error(..., NULL)`? Using an unknown aes is not a guaranteed usage (c.f. https://github.com/tidyverse/ggplot2/issues/3725#issuecomment-573716059), so you need to create a custom Geom that actually requires `groupS`, but I think it's a bit too heavy to test this issue.

Reply:
If you need some help, I'd be happy to provide the test code.